msgid ""
msgstr "Project-Id-Version: pyload\n"
"Report-Msgid-Bugs-To: 'dev@pyload.net'\n"
"POT-Creation-Date: 2013-11-30 16:17+0100\n"
"PO-Revision-Date: 2014-03-22 14:03-0400\n"
"Last-Translator: pyloadTeam <team@pyload.net>\n"
"Language-Team: Catalan\n"
"Language: ca_ES\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"
"X-Generator: crowdin.net\n"

#: pyload/plugins/network/curldownload.py:245
#, python-format
msgid "Download chunks failed, fallback to single connection | %s"
msgstr "Descàrrega per trossos fallida, tornant a la connexió única | %s"

#: pyload/plugins/base.py:327
msgid "Pil and tesseract not installed and no Client connected for captcha decrypting"
msgstr "Pil i tesseract no estan instal·lats i no hi ha cap Client connectat per desxifrar captchas"

#: pyload/plugins/base.py:331
msgid "No captcha result obtained in appropriate time."
msgstr "Cap resultat de captcha obtingut en el temps apropiat."

#: pyload/plugins/hoster.py:267 pyload/plugins/hoster.py:304
#: pyload/plugins/hoster/ard.py:79
#, python-format
msgid "Setting User and Group failed: %s"
msgstr "Aplicació d'Usuari i el Grup ha fallat: %s"

#: pyload/plugins/crypter.py:170
msgid "Not existing file or unsupported protocol"
msgstr "No existeix el fitxer o protocol no suportat"

#: pyload/plugins/crypter.py:228
#, python-format
msgid "Could not remove file '%s'"
msgstr "No ha pogut suprimir el fitxer \"%s\""

#: pyload/plugins/hoster/rapidsharecom.py:100
msgid "Rapidshare: Traffic Share (direct download)"
msgstr "Rapidshare: Tràfic Compartit (descàrrega directa)"

#: pyload/plugins/hoster/rapidsharecom.py:127
#: pyload/plugins/hoster/rapidsharecom.py:194
msgid "Already downloading from this ip address, waiting 60 seconds"
msgstr "Ja s'està descarregant des de aquesta adreça IP, esperant 60 segons"

#: pyload/plugins/hoster/rapidsharecom.py:131
msgid "Invalid Auth Code, download will be restarted"
msgstr "Codi d'Autenticació invàlid, la descàrrega serà reiniciada"

#: pyload/plugins/hoster/rapidsharecom.py:199
msgid "RapidShareCom: No free slots"
msgstr "RapidShareCom: No hi ha espais lliures"

#: pyload/plugins/hoster/rapidsharecom.py:202
msgid "You need a premium account for this file"
msgstr "Necessites un compte premium per aquest fitxer"

#: pyload/plugins/hoster/rapidsharecom.py:204
msgid "Filename reported invalid"
msgstr "Nom de fitxer informat invàlid"

#: pyload/plugins/hoster/rpnetbiz.py:26
#: pyload/plugins/hoster/realdebridcom.py:41
#: pyload/plugins/hoster/premium4me.py:28
#: pyload/plugins/hoster/multidebridcom.py:46
#: pyload/plugins/hoster/fastixru.py:36 pyload/plugins/hoster/zeveracom.py:22
#: pyload/plugins/hoster/unrestrictli.py:53
#: pyload/plugins/hoster/simplydebridcom.py:25
#: pyload/plugins/hoster/alldebridcom.py:38
#: pyload/plugins/hoster/rehostto.py:26 pyload/plugins/hoster/reloadcc.py:24
#: pyload/plugins/hoster/debriditaliacom.py:45
#: pyload/plugins/hoster/premiumizeme.py:22
#: pyload/plugins/addon/rpnetbiz.py:43
#, python-format
msgid "Please enter your %s account or deactivate this plugin"
msgstr "Si us plau, introdueixi  el seu compte de %s o desactivar aquest plugin"

#: pyload/plugins/hoster/meganz.py:56
msgid "Decryption failed"
msgstr "Desxifrat ha fallat"

#: pyload/plugins/hoster/meganz.py:106
msgid "No file key provided in the URL"
msgstr "Cap clau de fitxer proporcionat a la URL"

#: pyload/plugins/hoster/meganz.py:118
msgid "Error code:"
msgstr "Codi d'error:"

#: pyload/plugins/hoster/fileservecom.py:99
msgid "Parallel download error, now waiting 60s."
msgstr "Error de descàrrega paral·lela, esperant 60s."

#: pyload/plugins/hoster/fileservecom.py:215
msgid "Not logged in."
msgstr "No connectat."

#: pyload/plugins/hoster/uploadedto.py:129
msgid "API key invalid"
msgstr "Clau de API invàlida"

#: pyload/plugins/hoster/uploadedto.py:153
#, python-format
msgid "%s: Not enough traffic left"
msgstr "%s: No queda suficient tràfic"

#: pyload/plugins/hoster/uploadedto.py:156
msgid "Traffic exceeded"
msgstr "Tràfic excedit"

#: pyload/plugins/hoster/baseplugin.py:65
msgid "Authorization required (username:password)"
msgstr "Autorització requerida (usuari:contrasenya)"

#: pyload/plugins/hoster/netloadin.py:146
#: pyload/plugins/hoster/netloadin.py:170
msgid "File temporarily not available"
msgstr "Fitxer temporalment no disponible"

#: pyload/plugins/hoster/netloadin.py:183
#, python-format
msgid "Netload: waiting between downloads %d s."
msgstr "Netload: esperant entre descàrregues %d s."

#: pyload/plugins/hoster/netloadin.py:214
#, python-format
msgid "Netload: waiting for captcha %d s."
msgstr "Netload: esperant per captcha %d s."

#: pyload/plugins/hoster/netloadin.py:252
msgid "Downloaded File was empty"
msgstr "El fitxer descarregat estava buit"

#: pyload/plugins/hoster/filesmailru.py:99
#, python-format
msgid "There was HTML Code in the Downloaded File (%s)...redirect error? The Download will be restarted."
msgstr "Hi havia Codi HTML en el fitxer (%s) descarregat... error de re-direcció? Es re-iniciarà la descàrrega."

#: pyload/plugins/hoster/xhamstercom.py:74
#: pyload/plugins/hoster/xhamstercom.py:83
#: pyload/plugins/hoster/xhamstercom.py:86
#, python-format
msgid "long_url: %s"
msgstr "long_url: %s"

#: pyload/plugins/account.py:140 pyload/plugins/account.py:146
#, python-format
msgid "Could not login with account %(user)s | %(msg)s"
msgstr "No s'ha pogut iniciar sessió amb el compte %(user)s | %(msg)s"

#: pyload/plugins/account.py:141
msgid "Wrong Password"
msgstr "Contrasenya Errònia"

#: pyload/plugins/account.py:202
#, python-format
msgid "Get Account Info for %s"
msgstr "Obtenir Informació de la Compte per %s"

#: pyload/plugins/account.py:211
#, python-format
msgid "Error: %s"
msgstr "Error: %s"

#: pyload/plugins/account.py:264
#, python-format
msgid "Your Time %s has a wrong format, use: 1:22-3:44"
msgstr "L'hora %s està en un format incorrecte, empra: 1:22-3:44"

#: pyload/plugins/account.py:287
#, python-format
msgid "Account %s has not enough traffic, checking again in 30min"
msgstr "El compte %s no te suficient tràfic, es tornarà a comprovar d'aquí 30min"

#: pyload/plugins/account.py:295
#, python-format
msgid "Account %s is expired, checking again in 1h"
msgstr "El compte %s està caducat, es tornarà a comprovar d'aquí 1h"

#: pyload/plugins/account.py:312
#, python-format
msgid "Login with %s"
msgstr "Inicieu la sessió amb %s"

#: pyload/plugins/addon.py:163
#, python-format
msgid "Error executing addon: %s"
msgstr "Error executant el complement: %s"

#: pyload/plugins/account/bitsharecom.py:37
msgid "Activate direct Download in your Bitshare Account"
msgstr "Activa la descàrrega directa al teu compte de Bitshare"

#: pyload/plugins/crypter/serienjunkiesorg.py:130
msgid "Downloadlimit reached"
msgstr "Límit de descàrrega assolit"

#: pyload/plugins/addon/premium4me.py:29
msgid "Please add your premium.to account first and restart pyLoad"
msgstr "Si us plau, afegeiu el vostre compte premium.to i torneu a iniciar pyLoad"

#: pyload/plugins/addon/externalscripts.py:54
#, python-format
msgid "Installed scripts for %s: "
msgstr "Scripts instal·lats per %s: "

#: pyload/plugins/addon/externalscripts.py:69
msgid "Script not executable:"
msgstr "Script no executable:"

#: pyload/plugins/addon/externalscripts.py:80
#, python-format
msgid "Error in %(script)s: %(error)s"
msgstr "Error en %(script)s: %(error)s"

#: pyload/plugins/addon/captchatrader.py:69
#: pyload/plugins/addon/captcha9kw.py:59
#: pyload/plugins/addon/expertdecoders.py:50
#, python-format
msgid "%s credits left"
msgstr "%s crèdits restants"

#: pyload/plugins/addon/captchatrader.py:117
msgid "Could not send response."
msgstr "No s'ha pogut enviar la resposta."

#: pyload/plugins/addon/captchatrader.py:135
msgid "Your CaptchaTrader Account has not enough credits"
msgstr "El teu compte de CaptchaTrader no te crèdits suficients"

#: pyload/plugins/addon/captcha9kw.py:93
#, python-format
msgid "New CaptchaID from upload: %s : %s"
msgstr "Nou CaptchaID de càrrega: %s : %s"

#: pyload/plugins/addon/captcha9kw.py:129
msgid "Your Captcha 9kw.eu Account has not enough credits"
msgstr "El teu compte de 9kw.eu no te crèdits suficients"

#: pyload/plugins/addon/rehostto.py:32
msgid "Please add your rehost.to account first and restart pyLoad"
msgstr "Si us plau afegir el seu compte de rehost.to primer i reinicieu pyLoad"

#: pyload/plugins/addon/hotfolder.py:82
#, python-format
msgid "Added %s from HotFolder"
msgstr "Afegit %s des de HotFolder"

#: pyload/plugins/addon/clickandload.py:75
msgid "Click'N'Load: Port 9666 already in use"
msgstr "Click'N'Load: El Port 9666 ja s'està en ús"

#: pyload/plugins/addon/xmppinterface.py:83
#: pyload/plugins/addon/ircinterface.py:74
#, python-format
msgid "Package finished: %s"
msgstr "Paquet finalitzat: %s"

#: pyload/plugins/addon/xmppinterface.py:91
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s"
msgstr "Descàrrega finalitzada : %(name)s @ %(plugin)s"

#: pyload/plugins/addon/expertdecoders.py:95
msgid "Your ExpertDecoders Account has not enough credits"
msgstr "El teu compte de ExpertDecoders no te crèdits suficients"

#: pyload/plugins/addon/updatemanager.py:74
msgid "*** Plugins have been updated, please restart pyLoad ***"
msgstr "* * * Plugins s'han actualitzat, si us plau reinicieu pyLoad * * *"

#: pyload/plugins/addon/updatemanager.py:76
msgid "Plugins updated and reloaded"
msgstr "Plugins actualitzats i recarregats"

#: pyload/plugins/addon/updatemanager.py:79
msgid "No plugin updates available"
msgstr "No hi ha actualitzacions de plugins disponibles"

#: pyload/plugins/addon/updatemanager.py:96
msgid "No Updates for pyLoad"
msgstr "No hi ha actualitzacions del pyLoad"

#: pyload/plugins/addon/updatemanager.py:100
#, python-format
msgid "***  New pyLoad Version %s available  ***"
msgstr "***  La nova Versió %s de pyLoad està disponible  ***"

#: pyload/plugins/addon/updatemanager.py:101
msgid "***  Get it here: https://github.com/pyload/pyload/releases  ***"
msgstr "***  Obtengui'l aquí: https://github.com/pyload/pyload/releases  ***"

#: pyload/plugins/addon/updatemanager.py:104
msgid "Not able to connect server for updates"
msgstr "No es pot connectar al servidor per actualitzacions"

#: pyload/plugins/addon/updatemanager.py:148
#, python-format
msgid "New version of %(type)s|%(name)s : %(version).2f"
msgstr "Nova versió de %(type)s|%(name)s : %(version).2f"

#: pyload/plugins/addon/updatemanager.py:157
#: pyload/plugins/addon/updatemanager.py:162
#, python-format
msgid "Error when updating %s"
msgstr "S'ha produït un error mentre s'actualitzava %s"

#: pyload/plugins/addon/updatemanager.py:162
msgid "Version mismatch"
msgstr "La versió no coincideix"

#: pyload/plugins/addon/extractarchive.py:97
#, python-format
msgid "No %s installed"
msgstr "El %s no està instal·lat"

#: pyload/plugins/addon/extractarchive.py:99
#: pyload/plugins/addon/extractarchive.py:104
#, python-format
msgid "Could not activate %s"
msgstr "No s'ha pogut activar %s"

#: pyload/plugins/addon/extractarchive.py:109
msgid "Activated"
msgstr "Activat"

#: pyload/plugins/addon/extractarchive.py:111
msgid "No Extract plugins activated"
msgstr "No hi han connectors d'extracció activats"

#: pyload/plugins/addon/extractarchive.py:116
msgid "Extract package"
msgstr "Extreure el paquet"

#: pyload/plugins/addon/extractarchive.py:116
msgid "Scans package for archives and extract them"
msgstr "Analitza el paquet buscant arxius per extreure'ls"

#: pyload/plugins/addon/extractarchive.py:123
#, python-format
msgid "Package %s queued for later extracting"
msgstr "El paquet %s s'ha posat a la cua per una posterior extracció"

#: pyload/plugins/addon/extractarchive.py:146
#, python-format
msgid "Check package %s"
msgstr "Revisa el paquet %s"

#: pyload/plugins/addon/extractarchive.py:187
#, python-format
msgid "Extract to %s"
msgstr "Extreu a %s"

#: pyload/plugins/addon/extractarchive.py:202
msgid "No files found to extract"
msgstr "No s'han trobat fitxers per extreure"

#: pyload/plugins/addon/extractarchive.py:209
msgid "extracting"
msgstr "extraient"

#: pyload/plugins/addon/extractarchive.py:220
msgid "Password protected"
msgstr "Protegit amb contrasenya"

#: pyload/plugins/addon/extractarchive.py:241
msgid "Wrong password"
msgstr "Contrasenya incorrecta"

#: pyload/plugins/addon/extractarchive.py:249
#, python-format
msgid "Deleting %s files"
msgstr "Eliminant els fitxers %s"

#: pyload/plugins/addon/extractarchive.py:256
msgid "Extracting finished"
msgstr "Extracció finalitzada"

#: pyload/plugins/addon/extractarchive.py:262
msgid "Archive Error"
msgstr "Error d'arxiu"

#: pyload/plugins/addon/extractarchive.py:264
msgid "CRC Mismatch"
msgstr "El CRC no coincideix"

#: pyload/plugins/addon/extractarchive.py:268
msgid "Unknown Error"
msgstr "Error desconegut"

#: pyload/plugins/addon/extractarchive.py:320
msgid "Setting User and Group failed"
msgstr "Configuració d'usuari i el grup ha fallat"

#: pyload/plugins/addon/linkdecryptercom.py:37
msgid "Crypter list not found"
msgstr "Llista de Crypter no trobada"

#: pyload/plugins/addon/linkdecryptercom.py:51
msgid "Crypter list is empty"
msgstr "La llista de Crypter està buida"

#: pyload/plugins/addon/ircinterface.py:82
#, python-format
msgid "Download finished: %(name)s @ %(plugin)s "
msgstr "Descàrrega finalitzada: %(name)s @ %(plugin)s "

#: pyload/plugins/addon/ircinterface.py:95
#, python-format
msgid "New Captcha Request: %s"
msgstr "Nova petició de Captcha: %s"

#: pyload/plugins/addon/ircinterface.py:96
#, python-format
msgid "Answer with 'c %s text on the captcha'"
msgstr "Constesta amb 'c %s text del captcha'"

#: pyload/plugins/addon/premiumizeme.py:48
msgid "Please add a valid premiumize.me account first and restart pyLoad."
msgstr "Si us plau afegiu un compte vàlid premiumize.me primer i reinicieu pyLoad."

#: pyload/plugins/addon/captchabrotherhood.py:69
#, python-format
msgid "%d credits left"
msgstr "%d crèdits restants"

#: pyload/plugins/addon/multihoster.py:43
#, python-format
msgid "Activated %s"
msgstr "%s activat"

#: pyload/plugins/addon/multihoster.py:61
msgid "No Hoster loaded"
msgstr "No s'ha carregat cap proveïdor"
